CHANGELOG.md

```diff
@@ -1,5 +1,50 @@
 # Change Log
 
+## v1.4.0
+
+> Change log since v1.3.0
+
+### New Feature: JobSidecarTerminator
+
+In the Kubernetes world, it is challenging to use long-running sidecar containers for short-term application orchestration because there is no straightforward way to
```

Comment:
> In the Kubernetes world, it is challenging to use long-running sidecar containers for short-term application orchestration
It is challenging to use long-running sidecar containers for short-term job because...

> However, many sidecar containers may lack an active exit mechanism, and therefore, an additional solution is required to address this issue.
However most sidecar containers lack the ability to discovery the exit of main container.